aws/resource_aws_vpc_peering_connection.go

```diff
@@ -432,3 +381,7 @@ func checkVpcPeeringConnectionAvailable(conn *ec2.EC2, id string) error {
 	}
 	return nil
 }
+
+func isNoSuchVpcPeeringConnectionErr(err error) bool {
```

Comment:
Nitpick: We prefer to not introduce indirection in such simple cases.

Reply:
Removed.